### DISCUSSION TITLE
Privacy Manifest file PR merge

### DISCUSSION THREAD

**Discussion by rinkusadhwani** [Alamofire/Alamofire#3830] [General]
Hi,

I want to know by when are you planning to merge privacy manifest pull request which is in Open state now.

Thanks,
Rinku

**Comment by jshier** [comment]
When I get time to finalize the integration across all of the package managers and test its visibility when consumed in Xcode. Hopefully Apple gives us more concrete dates soon.

**Comment by rinkusadhwani** [comment]
Hi,

As i can see PR for manifest uodate is merged few minutes ago.

Can you please tell me release number which will have the chamges with
privacy file.

Thanks,
Rinku

**Reply by jshier** [reply]
Not yet, no. It will likely go out in the next release, 5.9, but that depends on if it works correctly and Apple finalizes the plans.

**Reply by macdrevx** [reply]
Looks like Apple's plans have solidified: https://developer.apple.com/news/?id=3d8a9yyh

Does that provide enough info to estimate a release date? Some guidance here will be helpful as we plan out the work to update our apps by May 1!
